files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml

```diff
@@ -1,105 +1,76 @@
 ---
-# ALL tags must be with dashes (-) instead of underscores (_)
+# NOTE: Use dashes (-) exclusively for tags and underscores (_) exclusively for destinations.
+# submit_request_cpus its called in pulsar and in plain condor only request_cpus
 destinations:
-  interactive_pulsar:
-    runner: pulsar_embedded
-    destination_name_override: "embedded_pulsar_interactive_c{cores}-m{mem}"
```

Comment:
Maybe we can think about a unified naming schema? The above is too long, but I could imagine something like:

## destination

* ep = embedded pulsar
* p = pulsar
* c = condor
* s = slurm
* k = k8s

## resolver

* d = docker
* s = singularity
* c = conda

## resources

* cXX = cores
* mYY = memory
* gZZ = GPUs
* n = network (e.g. interactive tools)

could lead to something like `{destination}_{resolver}_{resources}`